stylesheets/_utility.utilities.scss

```diff
@@ -75,19 +75,24 @@ p.success {
     background-color: color(gray, off-white);
     background-image: url($image-path + 'stripes.png');
     background-repeat: repeat;
+    margin: 0 auto;
     padding: 1em 0;
     text-align: center;
-}
-
-div.no-results {
```

Comment:
What the reason behind removing this? It's used in CXM in various places and will cause regression.

Reply:
https://github.com/jadu/pulsar/commit/da4472e0ce8cbd2392d2efd00f44559a1058f842#diff-d86f1c6d0c5c8ba3fe81b8d6c0a289a5

Reply:
ok, can you add a note to the description to cover this as I'll need to change this in a fair few places in CXM (breaking change).